fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -354,7 +354,7 @@ private String doBackwardPathOnly(final FedoraResource resource) {
         if (path != null) {
 
             if (resource instanceof NonRdfSourceDescription) {
-                path = path + "/" + FCR_METADATA;
+                path += "/" + FCR_METADATA;
```

Comment:
The Sonar recommendation is to use a StringBuffer instead of String concatenation. Your update is a different syntax for the same String concatenation.
In any case, the performance impact of String concatenation is tolerable here, and no update is needed.